napari/components/camera.py

```diff
@@ -54,6 +57,45 @@ def view_direction(self) -> np.ndarray:
         )
         return view_direction
 
+    @view_direction.setter
+    def view_direction(self, view_direction: Tuple[float, float, float]):
+        if (view_direction[0], view_direction[2]) == (0, 0):
+            up_direction = (-1, 0, 0)
+        else:
+            up_direction = (0, -1, 0)
+        self.set_view_direction(
```

Comment:
It might be handy to make the default up direction the current up direction. This would make it easier to rotate around an object.